engine/table/src/main/java/io/deephaven/engine/table/impl/MultiJoinTableImpl.java

```diff
@@ -318,66 +380,53 @@ private static Table doMultiJoinZeroKey(@NotNull final MultiJoinInput... joinDes
                     resultModifiedColumnSet,
                     redirections);
 
-            for (int ii = 0; ii < useDescriptors.length; ++ii) {
-                final MultiJoinInput jd = useDescriptors[ii];
-                if (jd.inputTable().isRefreshing()) {
-                    final QueryTable input = (QueryTable) jd.inputTable();
+            for (int ii = 0; ii < joinInputHelpers.length; ++ii) {
```

Comment:
There might be some code to re-use here with keyed.

Reply:
I'm not convinced it's worth the effort.